iheart-songs/src/song-detail.css

```diff
@@ -0,0 +1,43 @@
+.song-detail {
```

Comment:
it'd be better if we used .scss to scope the stylings properly which would be easier to read